Refresh to MM Docs: glossary, intro material (#286)
MetaMask/metamask-docs: docs/guide/sending-transactions.md

That means that transactions are always processed in order for a given account, and so incrementing nonces is a very sensitive matter that is easy to mess up, especially when a user is interacting with multiple applications with pending transactions using the same account, potentially across multiple devices.

For these reasons, MetaMask currently does not provide application developers any way to customize the nonce of transactions it suggests. Instead, MetaMask [assists the user in managing their transaction queue themselves](https://metamask.zendesk.com/hc/en-us/articles/360015489251).

### Gas Price [optional]

Optional parameter - best used on private blockchains.

> In Ethereum, the pool of pending transactions offers their gas price as a sort of tip to the block producers to include this transaction in a block in exchange for a transaction fee. That means high gas prices can mean faster processing, but also more expensive transactions.

Reviewer: @BboyAkers @mapachurro, I think this is a great use of "block producers", which emphasizes the function of validators/miners/etc. that is relevant to the topic in question. The sentence is still imperfect, however, because it's the individual transaction's gas price that counts as a single bid / tip, and it's compared to the gas prices on offer by the rest of the pending transaction pool. Might I suggest:

```suggestion
In Ethereum, every transaction specifies a price for the gas that it will consume. To maximize their profit, block producers will pick pending transactions with higher gas prices first when creating the next block. This means that a high gas price will usually cause your transaction to be processed faster, at the cost of greater transaction fees. Note that this may not be true for e.g. Layer 2 networks, which may have a constant gas price or no gas price at all.
```

To be combined with my other suggestions in this file.

> "MetaMask allows users to choose between "slow," "medium," and "fast" options for their gas price. To learn how to use advanced gas controls [visit here](https://metamask.zendesk.com/hc/en-us/articles/360022895972).

Reviewer: ```suggestion
In other words, while you can ignore this parameter on MetaMask's default networks, you may want to include it in situations where your application knows more about the target network than we do. On our default networks, MetaMask allows users to choose between "slow," "medium," and "fast" options for their gas price. To learn how to use advanced gas controls [visit here](https://metamask.zendesk.com/hc/en-us/articles/360022895972).
```

> We cannot know about the gas market on all blockchains because it requires some deep analysis. For this reason, while you can safely ignore this parameter on our main hosted networks, you may want to suggest a gas price in situations where your application knows more about the target network than we do.
> 

Reviewer: ```suggestion
```
